### PR TITLE
Wish List

### PR DESCRIPTION
**Pull request by KlaraAzimova**

### CODE REVIEW

File: src/main/java/kg/peaksoft/giftlistb6/api/WishApi.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/wish")
+@CrossOrigin(origins = "*", maxAge = 3600)
+@Tag(name = "Wish Api", description = "Wish List")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/giftlistb6/api/WishApi.java]: Please write description properly

File: src/main/java/kg/peaksoft/giftlistb6/api/WishApi.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+public class WishApi {
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/giftlistb6/api/WishApi.java:21]: Please use endpoint naming convention

File: src/main/java/kg/peaksoft/giftlistb6/api/WishApi.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import kg.peaksoft.giftlistb6.dto.responses.SimpleResponse;
+import kg.peaksoft.giftlistb6.dto.responses.WishResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/wishList")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/giftlistb6/api/WishApi.java]: camelCase?

File: src/main/java/kg/peaksoft/giftlistb6/dto/requests/WishRequest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import javax.persistence.Column;
+import java.time.LocalDate;
+import java.util.List;
+
+@Getter
+@Setter
+public class WishRequest {
+
+    private String wishName;
+
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/giftlistb6/dto/requests/WishRequest.java]: In request and response classes no need lines between fields

File: src/main/java/kg/peaksoft/giftlistb6/dto/responses/HolidayResponse.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class HolidayResponse {
+
+    private String name;
+
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/giftlistb6/dto/responses/HolidayResponse.java]: Same

File: src/main/java/kg/peaksoft/giftlistb6/dto/responses/WishResponse.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class WishResponse {
+
+    private Long id;
+
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/giftlistb6/dto/responses/WishResponse.java]: Same

**Review by i-Coderr — COMMENTED** [review]
Commented 21.10.22 at 12.30 am

File: src/main/java/kg/peaksoft/giftlistb6/dto/responses/HolidayResponse.java (hunk 5 above)
Comment (KlaraAzimova) [reply src/main/java/kg/peaksoft/giftlistb6/dto/responses/HolidayResponse.java:15]: Калганын Суннат жазат

**Comment by i-Coderr** [comment]
Resolve conflicts

**Pull request closed without merging**